src/Components/RosTable/RosTable.scss

```diff
@@ -0,0 +1,78 @@
+.ros-systems-table {
+  dl > .pf-c-description-list__group {
+    line-height: 0px;
+  }
+
+  .pf-link {
+    color: #2B9AF3;
+    font-weight: 600;
+
+    &:hover {
+      text-decoration: none;
+    }
+  }
+
+  .recommendations.green-400 {
+    color: #4CB140;
+  }
+
+  .recommendations {
+    padding-left: 25px;
+  }
+
+  .pf-c-progress {
+    grid-template-columns: auto;
+
+    .pf-c-progress__status {
+      text-align: left;
+    }
+  }
+}
+
+.progress-score-bar {
+  grid-gap: 5px;
+
+  span.pf-c-progress__status-icon {
+    display: none;
+  }
+
+  .pf-c-progress__bar {
+    &:before {
+      background: #B8BBBE;
+    }
+  }
+
+  .pf-c-progress__measure {
+    color: #6A6E73;
+  }
+}
+
+.green-100 {
+  .pf-c-progress__indicator {
+    background: #BDE2B9;
+  }
+}
+
+.green-400 {
+  .pf-c-progress__indicator {
+    background: #4CB140;
+  }
+}
+
+.gold-400 {
+  .pf-c-progress__indicator {
+    background: #F4C145;
+  }
+}
+
+.orange-300 {
+  .pf-c-progress__indicator {
+    background: #EC7A08;
+  }
+}
+
+.danger-200 {
+  .pf-c-progress__indicator {
+    background: #A30000;
+  }
+}
```

Comment:
I tried to import below line and Progress component -

```
import { Progress, ProgressVariant } from '@patternfly/react-core';
<Progress value={100} title="Title" variant={ProgressVariant.success} />
```

It was giving me error - `unable to import ProgressVariant` so I just used Progress component.
Any suggestion on how to pass different color other than `'danger' | 'success' | 'warning'` using variant?

Reply:
Mhh, I guess you are out of luck and have to either use classnames or ariables from `@patternfly/react-tokens`. Which you'll use is up to you, the tokens one would require setting style directly, which is not clean from CSS point, but all the logic would be in this file. With the classname you can use css vars for these as well, it will be cleaner from CSS side, but the logic would be spread in this file and in the stylesheet.